notebooks/tutorials/constant_potential.ipynb

```diff
@@ -523,11 +414,12 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 20,
+   "execution_count": 14,
    "id": "1fd1eb3e-41e6-4400-82c8-d86dcb29eba3",
    "metadata": {},
    "outputs": [],
    "source": [
+    "L_Z = pdb.topology.getPeriodicBoxVectors()[2][2]\n",
     "conp.setExternalField((0.0, 0.0, -POTENTIAL / L_Z))"
```

Comment:
Is it impossible to use constant potential with a barostat to let the box size change?  Since you specify the field, not the potential, any change in the box vectors would cause the potential to change.